src/components/FlyoutMenu/FlyoutMenu.jsx

```diff
@@ -0,0 +1,117 @@
+import './_flyout-menu.scss';
```

Comment:
This should be moved to the central sass entrypoint `./src/styles.scss`. We don't import styles directly in the component. The compiled sass is provided in the exported bundle and also provided to the storybook environment.